src/Build/Logging/BinaryLogger/BinaryLogRecordKind.cs

```diff
@@ -20,6 +20,9 @@ internal enum BinaryLogRecordKind
         ProjectEvaluationFinished,
         ProjectImported,
         ProjectImportArchive,
-        TargetSkipped
+        TargetSkipped,
```

Comment:
You also have to bump up the BinaryLogger.FileFormatVErsion: [https://github.com/cdmihai/msbuild/blob/2c7610b154bce223bfefadd147109aac2c631160/src/Build/Logging/BinaryLogger/BinaryLogger.cs#L20-L34](https://github.com/cdmihai/msbuild/blob/2c7610b154bce223bfefadd147109aac2c631160/src/Build/Logging/BinaryLogger/BinaryLogger.cs#L20-L34)

Reply:
Updated with latest commit.